### DISCUSSION TITLE
Add Comments to CSV export

### DISCUSSION THREAD

**Discussion by freezingDaniel** [iSoron/uhabits#1834] [Feature Requests] [answered]
While trying to find out for which reason I "fail" my habit most often I wanted to check and sum up the comments I leave on a specific habit.

Given that I did not want to click through every day manually I exported the CSV just to discover that the comments are not part of it.

Checkmarks.csv
is: `2023-09-17,2`
Expected: `2023-09-17,2,comment`

**Comment by 0ldb34r** [comment]
I agree. I don't understand the use of comments if i can't export them in the end.

**Comment by studgeek** [comment, marked as answer]
I believe this is a duplicate of #1567. So please comment/vote there also.
